src/backend/TrafficCourts/Staff.Service/Controllers/DisputeController.cs

```diff
@@ -35,7 +35,7 @@ public DisputeController(IDisputeService disputeService, ILogger<DisputeControll
     /// <param name="cancellationToken"></param>
     /// <returns>A collection of Dispute records</returns>
     [HttpGet("/disputes")]
-    [ProducesResponseType(typeof(ICollection<Dispute>), StatusCodes.Status200OK)]
+    [ProducesResponseType(typeof(IList<Dispute>), StatusCodes.Status200OK)]

```

Comment:
`ICollection<T>` probably is fine, but I was trying to be consistent with `IList<T>` when troubleshooting the output generation from swagger.